modules/subscriber/subscriber.tf

```diff
@@ -0,0 +1,100 @@
+data "aws_iam_policy_document" "subscriber" {
+  statement {
+    actions = [
+      "logs:DescribeSubscriptionFilters",
+      "logs:DeleteSubscriptionFilter",
+      "logs:PutSubscriptionFilter",
+      "logs:DescribeLogGroups",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "lambda:AddPermission",
+      "lambda:RemovePermission",
+    ]
+
+    resources = ["*"]
+  }
+}
+
+resource "aws_lambda_function" "subscriber" {
+  s3_bucket     = var.mode == "dev" ? "axiom-cloudformation-dev" : "axiom-cloudformation"
+  s3_key        = "axiom-cloudwatch-forwarder/v1.2.0/forwarder.zip"
```

Comment:
WDYT of interpolating the version here from a variable? That it would be easy to update the code to point to a new version when no Terraform changes are required.

Reply:
updated

Reply:
What I added is just replacing the version with:

```s3_key        = "axiom-cloudwatch-forwarder/v${var.forwarder_version}/forwarder.zip"```

Wondering if its better to replace the whole s3_key with a variable.